Platooning strategic plugin cannot be built with docker build

### Types of Issue

<!--- What types of issue? Replace the space with an `x` in the box that applies: -->

- [x] Anomaly report (something appears to not work correctly)
- [ ] Enhancement request (describe the enhancement being requested)
- [ ] Other (please ensure the description clarifies why the issue doesn’t fall into either of the above categories)

### Descriptive summary
When trying to build develop docker images the following error occurs
```
[ 85%] Generating C++ code from cav_srvs/SetLightBarIndicator.srv
[ 85%] Generating C++ code from cav_srvs/GetAvailableRoutes.srv
In file included from /home/carma/carma_ws/src/carma-platform/platooning_strategic/include/platoon_strategic.hpp:36:0,
                 from /home/carma/carma_ws/src/carma-platform/platooning_strategic/src/platoon_strategic.cpp:19:
/home/carma/carma_ws/src/carma-platform/platooning_strategic/include/state_machine.hpp:8:36: fatal error: cav_srvs/PlanManeuvers.h: No such file or directory
compilation terminated.
carma-platform/platooning_strategic/CMakeFiles/platoon_strategic_plugin_lib.dir/build.make:62: recipe for target 'carma-platform/platooning_strategic/CMakeFiles/platoon_strategic_plugin_lib.dir/src/platoon_strategic.cpp.o' failed
make[2]: *** [carma-platform/platooning_strategic/CMakeFiles/platoon_strategic_plugin_lib.dir/src/platoon_strategic.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```
### Expected behavior
No build errors on develop branch
### Software version this applies to
<!--- what version are you running that you noticed this problem or deficiency? -->
develop after version 3.3.0
Commit: https://github.com/usdot-fhwa-stol/carma-platform/commit/9a6b25b997463d27a0929e6414a684089a6ff9f7
### Actual behavior
Above error occurs
### Steps to reproduce the behavior
checkout develop
./build-image.sh -d


### Related work

Link to related tickets or prior related work here.

--- a/platooning_strategic/CMakeLists.txt
+++ b/platooning_strategic/CMakeLists.txt
@@ -24,6 +24,7 @@ set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
 find_package(catkin REQUIRED COMPONENTS
   carma_utils
   cav_msgs
+  cav_srvs
   roscpp
   std_msgs
   carma_wm
@@ -39,7 +40,7 @@ find_package(Boost REQUIRED COMPONENTS system)
 
 catkin_package(
    INCLUDE_DIRS include
-   CATKIN_DEPENDS carma_utils cav_msgs roscpp std_msgs carma_wm
+   CATKIN_DEPENDS carma_utils cav_msgs roscpp std_msgs carma_wm cav_srvs
 )
 
 ###########

--- a/platooning_strategic/package.xml
+++ b/platooning_strategic/package.xml
@@ -11,4 +11,5 @@
   <depend>roscpp</depend>
   <depend>std_msgs</depend>
   <depend>carma_wm</depend>
+  <depend>cav_srvs</depend>
 </package>

